src/client/pages/Offer/Introduction/Sidebar/index.tsx

```diff
@@ -225,7 +226,7 @@ export const Sidebar: React.FC<SidebarProps> = ({
   const isLoading =
     isLoadingQuoteCart || isLoadingCampaign || startDateProps.isLoading
   const isSwitcher = hasCurrentInsurer(offerData)
-  const isSwitchableForCar = false
+  const isSwitchableForCar = isCarSwitcher(offerData.quotes)
```

Comment:
You need a feature toggle here as well, for line 233 perhaps? And the comment from me can be updated. While we still don't have the `currentInsurer` field, we at least determine switcher status now, albeit in FE only.